Menus/MainMenuGUI.cpp

```diff
@@ -1142,8 +1142,7 @@ void MainMenuGUI::Update()
 					g_SettingsMan.SetShowToolTips(m_aOptionsCheckbox[SHOWTOOLTIPS]->GetCheck());
 					g_SettingsMan.SetPreciseCollisions(m_aOptionsCheckbox[PRECISECOLLISIONS]->GetCheck());
 
-					Writer writer("Base.rte/Settings.ini");
-                    g_SettingsMan.Save(writer);
+                    g_SettingsMan.UpdateSettingsFile();
```

Comment:
These changes are cause there was some convenience method made for this before that wasn't being used, right?
Also, the whitespace changes is cause of spaces -> tabs?

Reply:
Yeah I added this one a while back but forgot to use it in some places.